packages/nextjs/components/Tab.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useCallback, useRef, useState } from "react";
+import Image from "next/image";
+import Link from "next/link";
+
+const TabButton = ({
+  children,
+  tabIndex,
+  currentIndex,
+  onClick,
+}: {
+  children: React.ReactNode;
+  tabIndex: number;
+  currentIndex: number;
+  onClick: (index: number) => void;
+}) => {
+  return (
+    <button
+      onClick={() => onClick(tabIndex)}
+      className={` ${
+        tabIndex == currentIndex ? " bg-base-100" : "bg-base-300"
+      } btn btn-secondary btn-xl normal-case font-xl text-lg`}
+    >
+      {children}
+    </button>
+  );
+};
+
+/**
+ * Site header
+ */
+export const Tab = ({ onTabChange, currentIndex }: { onTabChange: (index: number) => void; currentIndex: number }) => {
```

Comment:
why do we need this tabs?

Reply:
https://www.figma.com/file/D0YZXzXhNtaP8j0lcq2JZR/Dice?type=design&node-id=1%3A2&mode=design&t=9nWbwZgT95vcsicq-1 This is  the ui its based on , and there a tab there